couchdb-cluster-admin/suggest_shard_allocation.py

```diff
@@ -42,5 +43,57 @@ def main(keep_shards_whole=True):
             print "  {}".format(shard)
 
 
+def get_db_size(node_details, db_name):
+    return get_db_metadata(node_details, db_name)['disk_size']
+
+
+def get_shard_names(node_details, db_name):
+    return sorted(get_shard_allocation(node_details, db_name).by_range)
+
 if __name__ == '__main__':
-    main()
+    from gevent import monkey; monkey.patch_all()
+    import gevent
```

Comment:
Since this is a util library and it would only be used once when instantiating hte database, is it worth adding more dependencies it to make it "fast"?